src/app/nav/navigate-store.js

```diff
@@ -24,13 +24,5 @@ var NavigateStore = createStore({
   },
   getCurrentPage: function () {
     return this.state.currentPage;
-  },
-  dehydrate: function () {
```

Comment:
I don't see any reason not to have their implementations be shared across all stores. A simple convention makes this possible.